packages/@aws-cdk/aws-ecs/test/fargate/test.fargate-task-definition.ts

```diff
@@ -111,6 +111,29 @@ export = {
         taskDefinition.addPlacementConstraint(ecs.PlacementConstraint.memberOf("attribute:ecs.instance-type =~ t2.*"));
       }, /Cannot set placement constraints on tasks that run on Fargate/);
 
+      test.done();
+    },
+
+    'throws when setting proxyConfiguration without networkMode AWS_VPC'(test: Test) {
```

Comment:
This test should not be here since this is `test.fargate-task-definition.ts` and you created an Ec2 task def. Also, for Fargate task def awsvpc will be the default and only networkMode.